src/components/core/core.test.js

```diff
@@ -55,14 +55,14 @@ describe('Core', function() {
     test('returns false when there\'s no active fullscreen element', () => {
       jest.spyOn(Fullscreen, 'fullscreenElement').mockReturnValue(undefined)
 
-      expect(this.core.isFullscreen()).toEqual(false)
+      expect(this.core.isFullscreen()).toBeFalsy()
```

Comment:
IMO, Core's `isFullscreen` method should only return boolean values. The use of `toBeTruthy` and `toBeFalsy` could give leeway for `undefined` values, so maybe we could use type coercion on the `isFullscreen` method and stick to the use of `toEqual`. 

AFAIK, `toEqual` helps avoid false positives and returns clearer messages on the terminal when a test fails.

What do you think?

Reply:
@pedrochamberlain  I agree with you, in this case the only thing I did was correct the tests for the last implementation that passed. We could implement this type coercion implementation in another pull request. What do you think?

Reply:
Sounds fair to me.

Reply:
Hi @oHervis, you can just undo your changes on `core.test.js` file and accept @pedrochamberlain suggestion for the `core.js` file. That way you solve this problem now and you don't need to open another PR.